precache/package.json

```diff
@@ -25,5 +25,6 @@
     "eslint-plugin-promise": "^5.1.0",
     "firebase-functions-test": "^0.3.0",
     "nodemon": "^2.0.7"
+    
```

Comment:
ฉันคิดว่าคุณไม่ได้ตั้งใจจะเปลี่ยนไฟล์นี้ คุณควรตรวจสอบการเปลี่ยนแปลงของคุณเมื่อคุณคอมมิต
I think you didn't mean to change this file. You should review your changes when you commit.